taichi/ir/ir.h

```diff
@@ -354,10 +353,9 @@ class StmtFieldManager {
   bool equal(StmtFieldManager &other) const;
 };
 
-#define TI_STMT_DEF_FIELDS(...) TI_IO_DEF(__VA_ARGS__)
+#define TI_STMT_DEF_FIELDS(...)
```

Comment:
The serialization declares for Stmt members seem never used but stopped me from compiling. Thus removed.